ompi/win/win.c

```diff
@@ -167,18 +167,18 @@ config_window(void *base, size_t size, int disp_unit,
                                      MPI_WIN_SIZE, size, true);
     if (OMPI_SUCCESS != ret) return ret;
 
-    ret = ompi_attr_set_fortran_mpi2(WIN_ATTR, win,
+    ret = ompi_attr_set_fortran_mpi1(WIN_ATTR, win,
                                      &win->w_keyhash,
                                      MPI_WIN_DISP_UNIT, disp_unit,
                                      true);
```

Comment:
I agree that `_mpi1()` is a more correct function to use here (because `disp_unit` is an `int`, not an `MPI_Aint`), but won't C automatically promote the `int` to `MPI_Aint`, and `_mpi2()` would work, too?

I.e., are you considering this a code cleanup, or is this fixing some type of actual problem?